crates/tauri-runtime-wry/src/lib.rs

```diff
@@ -2840,13 +2843,13 @@ impl<T: UserEvent> Runtime<T> for Wry<T> {
     let active_tracing_spans = self.context.main_thread.active_tracing_spans.clone();
     let proxy = self.event_loop.create_proxy();
 
-    self.event_loop.run(move |event, event_loop, control_flow| {
+    self.event_loop.run_return(move |e, event_loop, cf| {
```

Comment:
I renamed these variables here so that this stays on one line. That way, the git diff shows nicely that `run_return` is effectively what `run` used to be.